pkg/transponder/configure.go

```diff
@@ -82,6 +82,13 @@ func Configure(cmd *cobra.Command, args []string) error {
 		return errors.New("failed to list connections")
 	}
 
+	fmt.Println("Initiating data discovery")
+	fmt.Println("Delivering blueberries and pancakes")
```

Comment:
Nitpick.  I have heard in my previous life that customers sometimes find these things confusing.  I will leave it up to you on whether we should keep this or not.

Reply:
This is a product requirement to ship :) can take it up w Max